### PR TITLE
Kyle/symco 35 Excel View Changes

### PR DESCRIPTION
**Pull request by kaigoos**
Excel View:
- Changed download to be a button instead of a link
- Added label to change templates selector
- Search bar to search table for name, rating, or transcription contents
- Table header is sticky and will stay at the top of the page when scrolling down
- Page has horizontal scroll bar at the bottom if it doesn't fit in the window
- Minimum width of each transcription column is increased
- Added column to indicate if a conversation ended in a custom response
- Terminal nodes are indicated in the header
- Nodes ending in a custom response will have "(Custom Response") tagged on at the end of the transcription.

Response Table:
- Added column to indicate if a conversation ended in a custom response

### CODE REVIEW

**Comment by rosasck** [comment]
excel fails every once in a while during download... i dont get an error.. have you noticed this?

**Comment by kaigoos** [comment]
> 
> 
> excel fails every once in a while during download... i dont get an error.. have you noticed this?

I've never had it happen to me before. I'll try do some research and see what the issue might be.

**Review by Likhovodov — APPROVED** [review]
Looks good to me. I did not have issues downloading the spreadsheet. How keen was Tanya about audio in the excel? If she was we should explore that option

File: simcon_project/conversation_templates/views/template_responses_table.py [hunk 1]
```diff
@@ -36,31 +38,54 @@ def get(self, request, pk):
...
 
+        if 'filter' in request.GET:
+            table_data = filter_search(request, table_data)
+
         if not table_data:
             response_table = None
         else:
             response_table = ResponseTable(data=table_data, extra_columns=extra_columns)
+
+        RequestConfig(request, paginate=False).configure(response_table)
```
Comment (devin-w-hill) [inline simcon_project/conversation_templates/views/template_responses_table.py]: I get a django error here if the template doesn't have any responses and I go to the excel page.

**Review by devin-w-hill — CHANGES_REQUESTED** [review]
Everything works, just found one bug that should be easy to fix

**Comment by kaigoos** [comment]
> 
> 
> Looks good to me. I did not have issues downloading the spreadsheet. How keen was Tanya about audio in the excel? If she was we should explore that option

She didn't mention it when I talked to her about it so I'm not sure how important it is if it's in there or not

File: simcon_project/conversation_templates/views/template_responses_table.py (hunk 1 above)
Comment (kaigoos) [reply simcon_project/conversation_templates/views/template_responses_table.py]: Good catch!

**Review by ericharper1 — APPROVED** [review]

**Review by devin-w-hill — APPROVED** [review]

**Pull request merged**